yarn.lock

```diff
@@ -12031,14 +12031,16 @@ react-icons@^2.2.7:
   dependencies:
     react-icon-base "2.1.0"
 
-react-image-gallery@rentpath/react-image-gallery:
-  version "0.8.7"
-  resolved "https://codeload.github.com/rentpath/react-image-gallery/tar.gz/c6034ab83a54bc0b43abd8a424c367edeca77c21"
+react-image-gallery@0.8.9:
+  version "0.8.9"
+  resolved "https://registry.yarnpkg.com/react-image-gallery/-/react-image-gallery-0.8.9.tgz#7fc008443d54f0b8b97c208e12a2470d07e26413"
+  integrity sha1-f8AIRD1U8Li5fCCOEqJHDQfiZBM=
   dependencies:
     lodash.debounce "^4.0.8"
     lodash.throttle "^4.1.1"
     prop-types "^15.5.8"
-    react-swipeable "^4.0.0"
+    react-swipeable "^4.2.2"
+    resize-observer-polyfill "^1.5.0"
```

Comment:
interesting that we even obtained a new dependency. i wonder if this is a side effect of not versioning this thing to begin with

Reply:
We went from `0.8.7` to `0.8.9`, so maybe that did it?